pkgs/development/libraries/cosmopolitan/default.nix

```
@@ -0,0 +1,45 @@
+{ lib, stdenv, makeWrapper, runCommand, cosmopolitan }:
+stdenv.mkDerivation {
+  pname = "cosmopolitan-unstable";
+  version = "20210228";
+  src = builtins.fetchGit {
+    url = "https://github.com/jart/cosmopolitan";
+    rev = "94afa982c35d4322999b5591f6d41d5cfc3f19a5";
+    ref = "master";
+  };
+  postPatch = ''
+    sed -i -e "s|/usr/bin/env bash|/bin/sh|" build/findtmp
+  '';
+  dontConfigure = true;
+  dontFixup = true;
+  nativeBuildInputs = [ makeWrapper ];
```

```suggestion
  };

  postPatch = ''
    sed -i -e "s|/usr/bin/env bash|/bin/sh|" build/findtmp
  '';

  dontConfigure = true;
  dontFixup = true;

  nativeBuildInputs = [ makeWrapper ];
```

Why is the postPatch needed? bash should be available in the env. Also it should be converted to substituteInPlace.